app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+    # dictionary for saving links to class objects
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    dict_info_by_name = {}
+    # dictionary for saving full people information: name, age, marriage
```

Comment:
Comments shouldn't be present in the final version of your code

Reply:
and `Person.people` does the same for you

Reply:
How then do I know which attribute (husband or wife) to add a link to?